addons/l10n_in_website_sale/__manifest__.py

```diff
@@ -0,0 +1,8 @@
+{
+    'name': "Indian eCommerce",
+    'description': "Indian Localization E-Commerce Module",
+    'version': '1.0',
```

Comment:
```suggestion
    'version': '1.0',
    'countries': ['in'],
```
Since it's not an auto installed module